packages/web-components/src/components/rux-indeterminate-progress/rux-indeterminate-progress.scss

```diff
@@ -0,0 +1,89 @@
+:host {
+    --size: 3.75rem;
+    // These are used to determine nub-size
+    --wrapper-height: calc(var(--size) + 4px);
+    --second-div-height: calc(var(--size) * 0.9 - 6px);
+
+    --nub-size: calc(
+        ((var(--wrapper-height) - var(--second-div-height)) / 2) - 4px
+    );
+
+    // .5's here to pervent hard stops in mask
+    --mask: radial-gradient(
+        farthest-side,
+        transparent calc((((100% - 4px) * 0.9) - 5.5px)),
+        var(--color-background-interactive-default)
+            calc((((100% - 4px) * 0.9) - 4.5px))
+    );
+    display: flex;
```

Comment:
can we set this to inline-block by default? I think that'll be the most common use case, what do you think?

Reply:
Agreed! just changed it.